bucket/ludusavi.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "0.3.0",
+    "description": "Ludusavi is a tool for backing up your PC video game save data, written in Rust",
+    "homepage": "https://github.com/mtkennerly/ludusavi",
+    "license": {
+        "identifier": "MIT",
+        "url": "https://github.com/mtkennerly/ludusavi/blob/master/LICENSE"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/mtkennerly/ludusavi/releases/download/v0.3.0/ludusavi-v0.3.0-win64.zip",
+            "hash": "e74cc1088a6c1943b695071ed31ee5f0c6cc0dd81e40880c11485891936063d2"
+        },
+        "32bit": {
+            "url": "https://github.com/mtkennerly/ludusavi/releases/download/v0.3.0/ludusavi-v0.3.0-win32.zip",
+            "hash": "afbc374d5c5754ca590e82371472b45d4ab9f84ab8924a0c31a02ae0cb1f3567"
+        }
+    },
+    "bin": "ludusavi.exe",
+    "checkver": {
+      "github": "https://github.com/mtkennerly/ludusavi"
+    },
```

Comment:
Why the process is first spawning terminal when it is GUI application and why there is no icon??

```suggestion
    "shortcuts": [
        [
            "ludusavi.exe",
            "Ludusavi"
        ]
    ],
    "checkver": "github",
```

![A](https://i.imgur.com/EU8tqrA.gif)

Reply:
Hi, author here. The reason for the brief terminal is because it's both a GUI and a CLI application in one. I had a few options:

* Ideally, it wouldn't show a console at first and would just attach one to the process if necessary. Unfortunately, in my testing, that didn't seem to work correctly.
* It could leave the console open, but that leads to unnecessary screen clutter.
* So at least for now, the brief console that pops up seemed like the best compromise.

As for an icon, the GUI library I'm using doesn't yet support that in an official release, but once it does, I plan to add one: https://github.com/mtkennerly/ludusavi/issues/10 (there might be a way to add one sooner than that, but I need to try some tests first)